op-proposer/proposer/rpc/api.go

```diff
@@ -13,6 +13,7 @@ import (
 type ProposerDriver interface {
 	StartL2OutputSubmitting() error
 	StopL2OutputSubmitting() error
+	Propose(ctx context.Context, block *uint64) error
```

Comment:
nit: Timestamps (uint64) are used for super root proposals. We should use the more generic term, "sequence number". 
```suggestion
	Propose(ctx context.Context, seqNr *uint64) error
```